Basic-Scripts/Infix_to_Postfix_Conversion/Infix_to_Postfix_Conversion.py

```diff
@@ -0,0 +1,64 @@
+# function to convert infix to postfix using array as stack
+def infix_to_postfix(infix_expression):
+    """
+    Function to change infix expression to postfix one
+    Params:
+        infix_expression: Infix Expression provided by user
+    Returns:
+        postfix_expression: Postfix Expression convertef from Infix one
+    """
+
+    # initially empty stack
+    stack = []
+    # initially empty postfix_expression
+    postfix_expression = ""
+    for char in infix_expression:
+        # if an operand then put it directly in postfix expression
+        if char not in operators:
+            postfix_expression += char
+        # else if operators should be put in stack
+        elif char == "(":
+            """append function to push
+            elements in the stack"""
+            stack.append("(")
+        elif char == ")":
+            while stack and stack[-1] != "(":
+                postfix_expression += stack.pop()
+            """ pop function to pop
+                elements from stack in LIFO order """
+            stack.pop()
+        else:
+            """if priority of char in infix_expression is less than or equal to
+            char at stack[-1] pop out and put in postfix_expression"""
+            while (
+                stack and stack[-1] != "(" and priorities[char] <= priorities[stack[-1]]
+            ):
+                postfix_expression += stack.pop()
+            stack.append(char)
+    while stack:
+        postfix_expression += stack.pop()
+    return postfix_expression
+
+
+# Set of operators
+operators = set(["+", "-", "*", "/", "(", ")", "^"])
+
+# dictionary having priorities
+priorities = {"+": 1, "-": 1, "*": 2, "/": 2, "^": 3}
+
+print("Input")
+infix_expression = input("Enter infix expression\n")
+print("Output")
+
+# Displaying the Output
+print("Postfix expression: ", infix_to_postfix(infix_expression))
+
+"""
```

Comment:
Add this part in README under `Constraints`

Reply:
Updated changes. Kindly review.